Commands/presence.js

```diff
@@ -0,0 +1,10 @@
+module.exports = async function (client) {
+	client.user.setPresence({
```

Comment:
The indentation of the code is not correct. Please indent your code as shown in the image. Indent `client.user.setPresence`
<img src="https://i.imgur.com/sas0sw1.png">

Reply:
Yes Sure. On it!

Reply:
Yes Sure. On it!